tools/codegen/generate_config.yaml.template

```diff
@@ -0,0 +1,36 @@
+# =============================================
+# GENERATE_CONFIG.YAML TEMPLATE
+# =============================================
+# To use this template:
+# 1. Copy this file to generate_config.yaml
+# 2. Update the paths below with your local settings
+# 3. The generate_config.yaml file will be ignored by git
+# =============================================
+
+# ==============
+# CONFIGURE ONCE
+# ==============
+# Path to the Python package repository (required)
+# This is used both as the output directory for generated Python code
+# and as the repository path for updating pyproject.toml
+repo_path: /path/to/your/local/python/repo
```

Comment:
why is this needed? Isn't it just `../weave` or something?

Reply:
oh, this is not the weave repo - this is literally the new python repo. I would make this much more clear if so. why is this called `repo_path` and the others are `LANG_output`?

Reply:
fixed